pkg/common/constant.go

```diff
@@ -0,0 +1,10 @@
+package common
+
+const (
+	ErrorDetailsInfoLogLevel = iota
+	DefaultInfoLogLevel
+	StatesInfoLogLevel
+	StatesDetailsInfoLogLevel
+	DebugInfoLogLevel
+	TraceInfoLogLevel
```

Comment:
I don't think we need to have "Info" in all of them, and I'm not sure we need "LogLevel" either.

I know I don't want to type any of these ever anyway.